pyramid_oereb/core/processor.py

```diff
@@ -78,7 +98,10 @@ def plr_tolerance_check(self, extract):
                 if public_law_restriction.calculate(real_estate, Config.get('geometry_types')):
                     log.debug("plr_tolerance_check: keeping as potentially concerned plr {}".
                               format(public_law_restriction))
-                    inside_plrs.append(self.filter_published_documents(public_law_restriction))
+                    public_law_restriction = self.filter_documents_by_fosnr(public_law_restriction,
```

Comment:
I would add a little part to method doc that this is done. Otherwise it looks good.